chapters/file-handling/QN010.c

```diff
@@ -0,0 +1,49 @@
+/**
+ * Write a program to store a sentence from user in the file and display vowels
+ *  and consonants of the sentence.
+ */
+
+#include <stdio.h>
+
+int main()
+{
+  FILE *fptr, *fptr1, *fptr2;
+  char ch;
+
+  fptr = fopen("Alphabets.txt", "w");
+  printf("Enter a text:\n");
+  while ((ch = getchar()) != EOF)
+  {
+    putc(ch, fptr);
+  }
+
+  fclose(fptr);
+
+  fptr = fopen("Alphabets.txt", "r");
+  fptr1 = fopen("Vowels.txt", "w");
+  fptr2 = fopen("Consonants.txt", "w");
+  while ((ch = getc(fptr)) != EOF)
+  {
+    if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u')
+      putc(ch, fptr1);
+    else
+      putc(ch, fptr2);
+  }
```

Comment:
convert character to `lowercase` before comparison i.e. before `if` condition.
But while writing to the file, write the actual character from the input.